stories/RefreshCache.stories.js

```diff
@@ -0,0 +1,128 @@
+import React, { Component } from 'react';
+import { storiesOf } from '@storybook/react';
+import { InstantSearch, SearchBox } from '../packages/react-instantsearch/dom';
+import { CustomHits } from './util';
+
+const stories = storiesOf('RefreshCache', module);
+const buttonStyle = {
+  borderRadius: '4px',
+  padding: '10px',
+  border: 'none',
+  fontSize: '12px',
+  cursor: 'pointer',
+  color: '#fff',
+  background: '#3369e7',
+};
+
+const divStyle = {
+  borderRadius: '2px',
+  padding: '10px',
+  marginTop: '15px',
+  border: 'none',
+  fontSize: '12px',
+  color: '#999999',
+  background: '#F3F3F3',
+  display: 'block',
+};
+
+class AppWithRefresh extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      refresh: false,
+    };
+    this.refresh = this.refresh.bind(this);
```

Comment:
You could use the function property instead of `bind`, like you do with `onSearchStateChange`.